Jenkinsfile_CNP

```diff
@@ -30,7 +30,7 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 
 withPipeline("nodejs", product, component) {
 
-  if (env.CHANGE_TITLE && !env.CHANGE_TITLE.startsWith('[PREVIEW]')) {
+  if (env.CHANGE_TITLE && !env.CHANGE_TITLE.startsWith('[NO-CCD]')) {
```

Comment:
ok, we will need to monitor it and change it if we need to. I'm happy for this approach for now. Let's see how it works